features/step_definitions/electionSteps.js

```diff
@@ -18,6 +18,22 @@ module.exports = function() {
         this.election.save(callback);
     });
 
+    this.Given(/^I am on the retrieve vote page$/, function(callback) {
+        browser.get('/retrieve');
+        callback();
+    });
+
+    this.Given(/^I enter a random string as vote hash$/, function(callback) {
+        element(by.model('voteHash')).sendKeys('LAALLALALA');
+        callback();
+    });
+
+    this.When(/^I submit the form$/, function(callback) {
+        var button = element(by.css('button'));
```

Comment:
This'll fail if there's a button outside the form, should probably use `element(by.css('form'))` or `element(by.tagName('form'))` instead.